packages/@ember/-internals/glimmer/lib/modifiers/internal.ts

```diff
@@ -78,9 +78,7 @@ export class InternalModifierManager
     return null;
   }
 
-  getDebugInstance(): unknown {
-    return null;
-  }
+  abstract getDebugInstance(state: InternalModifierState): unknown;
```

Comment:
ah, this is what you meant

Reply:
Yeah, it’s a little bit of an edge case because we are implementing an interface. If we are extending an abstract super class you wouldn’t have to explicitly propagate it like this.